Makefile

```diff
@@ -2,39 +2,35 @@
 # Naming convention:
 #	for stable releases we use "1.0.0" format
 #   for pre-releases, we use   "1.0.0-beta.2" format
-VERSION=1.3.2-beta
+VERSION=1.5.0-alpha
```

Comment:
This will change later. I keep bumping version along the way from PR to PR. The released one will be 2.0